widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/HasOngoingSecureConversationUseCase.kt

```diff
@@ -22,12 +20,19 @@ internal class HasOngoingSecureConversationUseCase(
             secureConversationsRepository.unreadMessagesCountObservable,
             engagementStateUseCase()
         ) { pendingSecureConversations, unreadMessagesCount, state ->
-            isAuthenticatedUseCase() &&
-                (pendingSecureConversations || unreadMessagesCount > 0 || state is State.TransferredToSecureConversation)
+            pendingSecureConversations || unreadMessagesCount > 0 || state is State.TransferredToSecureConversation
```

Comment:
Removed the `isAuthenticatedUseCase()` check, as it was included in the `unread_messages_count` observable on the core SDK side.